src/Datadog.Trace/Tagging/AzureFunctionTags.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="AzureFunctionTags.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.ExtensionMethods;
+
+namespace Datadog.Trace.Tagging
+{
+    internal class AzureFunctionTags : InstrumentationTags
```

Comment:
I think this should be `AzureFunctionsTags` to be consistent with the product's branding.